hexrd/transforms/cpp_sublibrary/Makefile

```diff
@@ -0,0 +1,26 @@
+CXX = c++
+CXXFLAGS = -O3 -Wall -shared -std=c++20 -fPIC -funroll-loops -Wno-maybe-uninitialized
```

Comment:
I'm not sure if we can require C++20. We should verify that it is available in all of the versions of the compilers we want to support.

Reply:
I agree with this fully. In fact, I think I can just flip this to C++11 without any difference in behavior. This must have been a holdover from my own local testing, I'm happy to remove it!